main/gy521_imu.h

```diff
@@ -79,5 +79,17 @@ void setupIMU(){
     Serial1.print("Z Acceleration: ");
     Serial1.println(sample.accel.z);
     delay(1000);
-  }
+    imuReading imuSample = getIMU();
+    Serial1.print("X Gyro");
+    Serial1.println(imuSample.gyro.x);
+    delay(1000);
+    Serial1.print("Y Gyro");
+    Serial1.println(imuSample.gyro.y);
+    delay(1000);
+    Serial1.print("Z Gyro");
+    Serial1.println(imuSample.gyro.z);
```

Comment:
Could you also add logs for all of the accel values?

Reply:
They already are logs for accel values.